### DISCUSSION TITLE
Activity filter for original posts

### DISCUSSION THREAD

**Discussion by toffeebreath** [AprilSylph/XKit-Rewritten#714] [Suggestion Box]
I would love to be able to filter my activity page so that I only see notes on posts that I have made, and hide notes on posts that I have reblogged.

**Comment by marcustyphoon** [comment]
Ooh. I want this too. Will have to check how viable it is.

Not sure where it would go in the extension! Feels sort of like Show Originals meets NotificationBlock... as a tweak...?

Edit: Did some work on the back end #715 if we want to pursue this.

**Comment by JRHB314** [comment]
I think this is a good idea; right now I manually tag all my original posts so that I can find and check them later, but for someone that has more original posts, that would get a bit tedious.

**Comment by alleycatboy** [comment]
@marcustyphoon I can look into picking up that branch and pursuing it!
